.ci/deps.sh

```diff
@@ -55,8 +55,10 @@ done
 python setup.py --help
 
 # Dart Lint commands
-wget -nc -O ~/dart-sdk.zip https://storage.googleapis.com/dart-archive/channels/stable/release/1.14.2/sdk/dartsdk-linux-x64-release.zip
-unzip -n ~/dart-sdk.zip -d ~/
+if [ ! -d "~/dart-sdk/" ] && [ ! -d "~/dart-sdk/bin" ]; then
```

Comment:
Would it not be better to do `if ! dartlint -v ; then` as it will also check if the dartlint is installed correctly ?